crates/extension/src/extension_store.rs

```diff
@@ -772,6 +772,7 @@ impl ExtensionStore {
 
     pub fn uninstall_extension(&mut self, extension_id: Arc<str>, cx: &mut ModelContext<Self>) {
         let extension_dir = self.installed_dir.join(extension_id.as_ref());
+        let work_dir = self.wasm_host.clone().work_dir.join(extension_id.as_ref());
```

Comment:
We don't need to `clone` here:

```suggestion
        let work_dir = self.wasm_host.work_dir.join(extension_id.as_ref());
```

Reply:
done!